### PR TITLE
Update pre-commits

### PR DESCRIPTION
**Pull request by jarrodmillman**

### CODE REVIEW

**Comment by larsoner** [comment]
... looks like it worked:

https://results.pre-commit.ci/run/github/11639513/1692286665.X309Hbw3TWq2shr-Hwn_hA

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.4.0
+    rev: f71fa2c1f9cf5cb705f73dffe4b21f7c61470ba9 # frozen: v4.4.0
```
Comment (larsoner) [inline .pre-commit-config.yaml:6]: wait... why freeze to specific commits? Shouldn't we let these update and make incremental changes as new bugs etc. are found? Usually the bot will do the heavy lifting for us

Comment (jarrodmillman) [reply .pre-commit-config.yaml:6]: I just ran `$ pre-commit autoupdate --freeze`. The bot should do it as well, I normally fix the rev and have the bot update monthly to keep down the noise.

Comment (larsoner) [reply .pre-commit-config.yaml:6]: Okay, feel free to configure the bot as you see fit now that it's enabled!

**Pull request merged**